activestorage/test/fixture_set_test.rb

```diff
@@ -4,11 +4,6 @@
 require "database/setup"
 
 class ActiveStorage::FixtureSetTest < ActiveSupport::TestCase
-  self.fixture_path = File.expand_path("fixtures", __dir__)
-  self.use_transactional_tests = true
-
-  fixtures :all
```

Comment:
Keeping this line within a particular test was a mistake. If the test wasn't loaded before the `activestorage/test/models/attachment_test.rb` file, ID would start at `1`, and so those tests would continue to pass.

Reply:
While this might have contributed to the cause, it turns out that moving this out to the `test/test_helper.rb` file breaks many other tests, so in the interest of fixing CI, I've reverted the move.